templates/openstack/install.html

```diff
@@ -39,54 +38,82 @@ <h1>Install OpenStack yourself</h1>
 </section>
 
 <section class="p-strip--light is-bordered">
-  <div class="row">
-    <div class="col-8">
-      <h2>Single-node, multi-node or large scale?</h2>
-      <p><strong>Choose the OpenStack installation option that suits you best:</strong></p>
+  <div class="row u-equal-height">
```

Comment:
Only col's should be direct children of rows, you can change this to a u-fixed-width or nest a col-12 within with everything else in that.

Reply:
Thanks, I was actually looking for this class. Got it wrong.